recipes/vibration.py

```diff
@@ -50,8 +51,11 @@ class VibrationsJob(MultiJob):
     """
     _result_type = VibrationsResults
 
-    def __init__(self, molecule, settings, jobType=ADFJob, get_gradients='get_gradients', reorder='inputOrder', aseVibOpt={}):
-        MultiJob.__init__(self)
+    def __init__(self, molecule, settings, name='plams.vib', jobType=ADFJob, get_gradients='get_gradients', reorder='inputOrder', aseVibOpt={}):
+
+        def _dummy_reorder(ignore, data):
+            return data
+        MultiJob.__init__(self, name=name)
```

Comment:
```suggestion
        super().__init__(self, name=name)
```
Style nitpick, feel free to ignore.